src/utils/re_frame.cljs

```diff
@@ -76,6 +77,14 @@
     (swap! handler-nesting-level dec)
     res))
 
+(defn delay-render
+  [content]
+  (let [render? (reagent/atom false)]
+    (js/setTimeout #(reset! render? true) 0)
+    (fn []
+      (when @render?
+        content))))
```

Comment:
Why is this in re-frame utils ns?

Reply:
idk, do you think it deserves a separate namespace?

Reply:
Just think re-frame is not involved/related to this component.

Probably in a utils namespace it fits better. If we have more components like this (performance focused) then we could add a new ns.

Reply:
yes we can move it no problem, i agree it's not related to re-frame, just for some reason i decided to put it there